### PR TITLE
Remove kubeless

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- Remove `kubeless` chart and tests
- Turn off all tests that use `kubeless`
- Remove `kubeless` from all MDs

**Related issue(s)**

[Issue #7736](https://github.com/kyma-project/kyma/issues/7736)

### CODE REVIEW

File: tests/end-to-end/external-solution-integration/cmd/fakelambda/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Fake Lambda
+
+## Overview
+
+This application was prepared to behave like the kubeless lambda and is used in `pkg/testsuite/deploy_fake_lambda.go` file.
```
Comment (kazydek) [inline tests/end-to-end/external-solution-integration/cmd/fakelambda/README.md:5]:
```suggestion
This application was prepared to behave like the kubeless lambda and is used in the `pkg/testsuite/deploy_fake_lambda.go` file. 
```

File: tests/end-to-end/external-solution-integration/cmd/fakelambda/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Fake Lambda
+
+## Overview
+
+This application was prepared to behave like the kubeless lambda and is used in `pkg/testsuite/deploy_fake_lambda.go` file. 
+
+>**NOTE:** Fake Lambda should be removed after migration to serverless v2
```
Comment (kazydek) [inline tests/end-to-end/external-solution-integration/cmd/fakelambda/README.md:7]:
```suggestion
>**NOTE:** Fake Lambda should be removed after migration to Serverless v2.
```

File: tests/end-to-end/upgrade/main.go [hunk 3]
```diff
@@ -101,14 +98,11 @@ func main() {
...
-	kubelessCli, err := kubeless.NewForConfig(k8sConfig)
-	fatalOnError(err, "while creating Kubeless clientset")
-
 	domainName, err := getDomainNameFromCluster(k8sCli)
 	fatalOnError(err, "while reading domain name from cluster")
 
-	kymaAPI, err := kyma.NewForConfig(k8sConfig)
-	fatalOnError(err, "while creating Kyma Api clientset")
+	//kymaAPI, err := kyma.NewForConfig(k8sConfig)
+	//fatalOnError(err, "while creating Kyma Api clientset")
```
Comment (Szymongib) [inline tests/end-to-end/upgrade/main.go:105]: Should the comments be there? There are a few more in this file

File: resources/core/templates/tests/test-external-solution.yaml [hunk 4]
```diff
@@ -26,7 +26,7 @@ spec:
         - ip: {{ $.Values.global.minikubeIP }}
           hostnames:
             - "connector-service.{{ $.Values.global.ingress.domainName }}"
-            - "counter-service-{{ $val.testsuite }}.{{ $.Values.global.ingress.domainName }}"
+            - "ctr-svc-{{ $val.testsuite }}.{{ $.Values.global.ingress.domainName }}"
```
Comment (Szymongib) [inline resources/core/templates/tests/test-external-solution.yaml:29]: What is the reason for changing this name? I feel like `ctr-svc` does not tell much for someone who is not familiar with the code.

Comment (pPrecel) [reply resources/core/templates/tests/test-external-solution.yaml:29]: `counter-service` name were too long and sometimes the e2e tests `compass-runtime-agent-adapter-external-solution`

File: tests/end-to-end/upgrade/main.go (hunk 3 above)
Comment (kubadz) [inline tests/end-to-end/upgrade/main.go:105]: Please remove these two lines.

File: tests/end-to-end/upgrade/chart/upgrade/templates/rbac.yaml [hunk 5]
```diff
@@ -80,12 +80,9 @@ rules:
...
 
-# Kubeless
-- apiGroups: ["kubeless.io"]
-  resources: ["functions", "httptriggers", "cronjobtriggers"]
-  verbs: ["get", "list", "watch", "update", "create", "delete"]
-- apiGroups: ["gateway.kyma-project.io"]
-  resources: ["apis", "apirules"]
+# Gateway
+- apiGroups: ["gateway.kyma-project.io"]	
+  resources: ["apis", "apirules"]
```
Comment (kubadz) [inline tests/end-to-end/upgrade/chart/upgrade/templates/rbac.yaml:85]: Please remove `"apis"`.

File: tests/end-to-end/upgrade/main.go (hunk 3 above)
Comment (pPrecel) [reply tests/end-to-end/upgrade/main.go:105]: Good point!

**Pull request merged**